blockchain-source/src/cardano/time.rs

```diff
@@ -15,27 +16,31 @@ impl Era {
         start_epoch: 208,
         known_time: 1596059091,
         slot_length: 1,
+        epoch_length_seconds: 432000,
     };
 
     pub const SHELLEY_TESTNET: Self = Self {
         first_slot: 1598400,
         start_epoch: 74,
         known_time: 1595967616,
         slot_length: 1,
+        epoch_length_seconds: 432000,
```

Comment:
It cannot change, so we can leave this hardcoded, right?

Reply:
yes

Reply:
alright, thanks for clarifying